tfac/figures/figureS1.py

```diff
@@ -0,0 +1,61 @@
+"""
+Figure to show each components largest weights for CCLE Data, Component 1 
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+from .common import subplotLabel
+from tensorpack import perform_CP
+from tensorpack.cmtf import cp_normalize
+from ..dataHelpers import import_LINCS_CCLE, proteinNames, reorder_table
+
+def getsetup(figsize):
+    sns.set(style="whitegrid", font_scale=0.7, color_codes=True, palette="colorblind", rc={"grid.linestyle": "dotted", "axes.linewidth": 0.6})
+    fig = plt.figure(figsize=figsize, constrained_layout=True)
+
+    gs = fig.add_gridspec(2, 5)
+    ax1 = fig.add_subplot(gs[0, 0])
+    ax2 = fig.add_subplot(gs[0, 1])
+    ax3 = fig.add_subplot(gs[0, 2])
+    ax4 = fig.add_subplot(gs[0, 3])
+    ax5 = fig.add_subplot(gs[0, 4])
+    
+    ax7 = fig.add_subplot(gs[1, 0])
+    ax8 = fig.add_subplot(gs[1, 1])
+    ax9 = fig.add_subplot(gs[1, 2])
+    ax10 = fig.add_subplot(gs[1, 3])
+    ax11 = fig.add_subplot(gs[1, 4])
+
+    return ([ax1, ax2, ax3, ax4, ax5], [ax7, ax8, ax9, ax10, ax11], fig)
```

Comment:
why not use the `getSetup()` from the common.py?

Reply:
I think I saw Figure2 use its own, It seemed easier at first to make 2/3 lists of axes instead of one large list but I can change it to use from common.py

Reply:
Please change it to use the one from common.py. 
I had to define a getsetup for figure 2 because the sizes of subplots I wanted to use for the ones in the first row were different from the second and third row.

Reply:
Fixed